__mocks__/vscode.ts

```diff
@@ -399,6 +399,13 @@ export namespace workspace {
    */
   export let rootPath: string | undefined;
 
+  export function getConfiguration(
+    section?: string | undefined,
+    scope?: any | null
+  ): undefined {
+    return undefined;
+  }
+
```

Comment:
This is not used anywhere, I guess we don't needed.

Reply:
It's a mock for tests, and some of the functions which call it are called all over the place...I'd rather leave it...

I changed the first thing though.